native/native_image_options.go

```diff
@@ -0,0 +1,94 @@
+package native
+
+import (
+	"fmt"
+	"github.com/magiconair/properties"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type NativeImageOption func(*options) error
```

Comment:
We have used this options pattern in other places in the code but implemented it in a functional way, see [here](https://github.com/paketo-community/cargo/blob/main/cargo/cargo.go#L35-L44) as an example.

I like that approach and would prefer that approach as it results in fewer objects created & fewer structs around. Unless you have specific intentions for doing things this way, something that wouldn't work using the more functional approach.

Reply:
I liked the functional approach and I will happy to do the refactor, in this case, because I was only using 1 option parameter I kept the static way of handling the parameters. I can work on the refactoring on another PR